getting-started/000-getting-started.md

```diff
@@ -1,16 +1,20 @@
 Get started with Mapzen Search
 
-Mapzen Search is a modern geocoding and place search service based on [Pelias](https://github.com/pelias/pelias), an experimental open geocoder from [Mapzen](https://mapzen.com) and powered by open data. It's used to transform the way people talk about places - with names and addresses - into geographic coordinates.
+Mapzen Search is a modern geocoding and place search service based on [Pelias](https://github.com/pelias/pelias), an experimental, open geocoder from [Mapzen](https://mapzen.com) and powered by open data. It's used to transform the way people talk about places - with names and addresses - into geographic coordinates, and to translate geographic coordinates into recognizable names of places.
 
-[REWRITE]
 Mapzen Search can be used to:
 - geocode addresses, points of interest, neighbourhood, cities, and regions
 - power real-time search autocomplete for places
-- turn a list of addresses into coordinates to plot
+- turn a list of addresses into coordinates that can be plotted on a map or computed
 - Take location coordinates and turn them into human readable places, like an address
 - find the neighbourhood, cities, states, and countries of location coordinates to quickly classify where your data is coming from
```

Comment:
something like this? I'm imagining being a CTO reading this ;)

> - quickly localize your users so they can discover locally relevant content.
> - provide a super-fast typeahead solution which feels intuitive for non technical users.
> - improve your database by adding geographic columns, such as `home_city` and `country` to your users table.
> - store numeric co-ordinate data instead of strings, for later use in reporting, analytics and machine learning.